### ISSUE TITLE
Add QR code && make yg.gl address always visible

### ISSUE THREAD

**Issue by denilsonsa** [RobinLinus/snapdrop#38]
Simple suggestion: add a QR code to the webpage. This way, people can easily share the address without having to type anything. It can probably be at the top-left of the page (which is empty now). Tapping on it might expand it (similar to how the "?" button works), to make it easier to scan.

![image](https://cloud.githubusercontent.com/assets/121676/21466028/df209766-c9b9-11e6-9c0b-a41308e4a533.png)

(created using <https://zxing.appspot.com/generator/>)

Also, after at least two devices are connected, the "yg.gl" short address disappears from the page. So, it is hard to remember and to tell people which address they should type. Thus, maybe "yg.gl" could be shown at the top of the page.

Thanks!

**Comment by jrruethe** [comment]
I agree with both of these feature requests, they would be very handy!

**Comment by Chi7519** [comment]
That makes **A LOT** of sense, I am just seeing the GitHub page for the first time, and I just wanted to see how it works... I looked at this and I love it... this would be a great accessory for mobile users! Best part is that it would make the app more accessable! 👍 😄

**Comment by Dtrieb** [comment]
+1  👍
